clang/test/dpct/cmake_migation/case_017/input.cmake

```diff
@@ -0,0 +1,16 @@
+# In the following cases check for changes after running the pattern-rewritter
+set (CMAKE_CXX_STANDARD g++)
```

Comment:
This is an invalid cmake syntax. 
Pls make sure the cmake file contains correct syntax.